contents/security/crossSiteScripting/crossSiteScripting.md

```diff
@@ -9,64 +9,87 @@
 
-This means that any malicious users (i.e. hackers) can write a script on their comments, such that everyone that visits the website will have the script executed. The script can be malicious in nature (in our example, the script sends the cookie of the visitor to the hacker).
+<modal title="**What is a Cookie?**" id="modal:cookie">
+A cookie is a piece of data sent from the server and stored on the client's computer. It can contain sensitive information such as login data. You can read more about it <a href="https://en.wikipedia.org/wiki/HTTP_cookie" target="_blank">here</a>.
+</modal>
 
-Therefore, the hacker has managed to add additional malicious "functionality" to the website, that is not intended by the original website developer.
+Therefore, the malicious user has managed to add additional "functionalities" to the website that is not intended by the original website developer.
 
 > Trivia: The term "Cross Site Scripting" is actually an old term. It originally describe an attack whereby hackers write malicious JavaScript scripts on a separate website, and injects it into the victim's website, in order to steal data from the victim's website/deface the page (hence "Cross Site"). Such attacks are no longer possible today, and the modern definition of "Cross Site Scripting" includes attacks that do not need to be on a separate website to work.
 
-## Types of Cross Site Scripting
+## Types of XSS
 
 There are no standard definitions, but there are at least two different types of
 XSS:
 
-* Persistent XSS (also known as stored XSS) - The JavaScript code gets stored in
-the web server. Example: Posting a malicious JavaScript code into a blog post as
-a comment. The comment gets stored in the database, and the web browser of future visitors of the
-webpage will retrieve the comment from the database, resulting in execution.
+* Persistent XSS (also known as stored XSS) - The injected `JavaScript` code gets stored in
+the web server. Example: Posting a malicious `JavaScript` code into a blog post as
+a comment. The comment gets stored in the server database, and when visitors visit the webpage, their web browsers will retrieve the comment from the database and run the malicious `JavaScript` code automatically.
 
-* Non-persistent XSS (also known as reflected XSS) - The JavaScript code is
+* Non-persistent XSS (also known as reflected XSS) - The `JavaScript` code is
 inserted in URL/links of website that accepts URL parameters as input. The input
-is therefore not stored in a database.
+is not stored in the server database.
 
 An example of a non-persistent XSS attack would be an e-card website that displays an e-card to a visitor of the website. The e-card can be customised by modifying the `content` parameter of the URL:
```

Comment:
This part should appear at part of the second bullet point?

Reply:
Indented